sdk/dapp-kit/src/walletStore.ts

```diff
@@ -56,7 +58,7 @@ export function createWalletStore({
 			(set, get) => ({
 				autoConnectEnabled,
 				wallets,
-				accounts: [],
+				accounts: [] as WalletAccount[],
```

Comment:
is this cast needed? I noticed the types blow up sometimes in my IDE, but I didn't think there was an actual typing issue lol